modules/programs/pqiv.nix

```diff
@@ -41,6 +39,26 @@ in {
         };
       '';
     };
+
+    extraConfig = mkOption {
+      type = types.lines;
+      default = "";
+      description = ''
+        Extra lines to be added to <filename>$XDG_CONFIG_HOME/pqivrc</filename> See <link
+        xlink:href="https://github.com/phillipberndt/pqiv/blob/master/pqiv.1"/>
+        for a list of available options.
```

Comment:
Should be Markdown
```suggestion
        Extra lines to be added to {file}`$XDG_CONFIG_HOME/pqivrc`. See
        {manpage}`pqiv(1)` for a list of available options.
```

Reply:
Ok, I've also edited the description for the other `settings` option as well.